internal/cli/command/cluster/create.go

```diff
@@ -108,12 +108,17 @@ func runCreate(banzaiCli cli.Cli, options createOptions) error {
 	}
 
 	log.Debugf("create request: %#v", out)
-	cluster, _, err := banzaiCli.Client().ClustersApi.CreateCluster(context.Background(), orgID, out)
+	cluster, resp, err := banzaiCli.Client().ClustersApi.CreateCluster(context.Background(), orgID, out)
 	if err != nil {
 		cli.LogAPIError("create cluster", err, out)
 		return errors.WrapIf(err, "failed to create cluster")
 	}
 
+	if resp.StatusCode == http.StatusCreated {
+		log.Infof(`cluster "%s" is already being created`, out["name"])
```

Comment:
**Question**: wouldn't the `cluster is being created` message from below be more consistent to use here? If the CLI log is 
```
cluster xyz is already being created
cluster xyz is already being created
cluster is being created
cluster is being created
```

I think that is a worse user experience than

```
cluster is being created
cluster is being created
cluster is being created
cluster is being created
```

What do you think?

Reply:
I think because the function returns after logging `cluster xyz is already being created`, the mixed response can not occur if I am not missing something...

Reply:
You are right about the function returning, but the multiple `cluster is being created` message is a thing as of now, so the function might be called multiple times maybe?

I remember we tested the issue where the PKE cluster creation could not query the cluster for 10-30 seconds after creation, because the secret was missing there.
IIRC I have seen multiple `cluster is being created` messages there.

Reply:
I'm fine with it this way anyway, just wanted to point it out, we can leave it as is.

Reply:
I meant [this](https://github.com/banzaicloud/banzai-cli/pull/354) one above.

Reply:
Yeah, so that part comes after the shortcut you introduced, so it's going to be fine, sorry about the holdup.

Reply:
The two issue is not related, the one you mentioned handles when you first create a cluster with the `wait` flag, this one is about when you create a cluster with the same name with the `wait` flag the second time while the first cluster creation is still running. So it is not possible to get a `cluster is being created` log message (in context of the same cluster) after you got a `cluster xyz is already being created` is what I am trying to say.